### PR TITLE
Implement the Addon for Google PubSub Writer

### PR DESCRIPTION
**Pull request by nintran52**
### What does this do?
- Add the Addon for Google PubSub Writer to receive consumer data from MQTT-broker and push to Google PubSub, then Google PubSub writes data to BigQuery
Screenshot 2023-11-21 at 08 34 00

### CODE REVIEW

File: docker/addons/pubsub-writer/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# inspection and data visualization.
+
+version: "3.7"
+
+networks:
+  mainflux-base-net:
+
+services:
+  pubsub-writer:
+    image: ion-broker/mainflux/pubsub-writer:${MF_RELEASE_TAG}
```
Comment (nnpvaan) [inline docker/addons/pubsub-writer/docker-compose.yml:18]: should change to: 
`${IMAGE_PROJECT}/${MF_DOCKER_IMAGE_NAME_PREFIX}/pubsub-writer:${MF_RELEASE_TAG}`

**Review by nnpvaan — CHANGES_REQUESTED** [review]
I think we need to add one more step to build pubsub-writer image in github action

File: cmd/pubsub-writer/main.go [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	}
+
+	client, err := pubsub.NewClient(ctx, cfg.ProjectID)
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to create client: %s", err))
+	}
+	defer client.Close()
+	topic, err := writerps.CreateTopic(ctx, client, cfg.TopicID)
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to create topic: %s", err))
```
Comment (wilianto) [inline cmd/pubsub-writer/main.go:85]: should be fatal here.

**Review by wilianto — APPROVED** [review]
lgtm! just left a minor panic comment.

**Comment by nintran52** [comment]
> I think we need to add one more step to build pubsub-writer image in github action

Like other add-ons, we just add the image name on the SERVICES (Makefile) which we need to build image

**Review by nnpvaan — APPROVED** [review]
LGTM

**Pull request merged**